paella-matterhorn/plugins/edu.harvard.dce.infoButtonPlugin/dce_infobutton.js

```diff
@@ -51,7 +51,13 @@ Class ('paella.plugins.InfoPlugin', paella.ButtonPlugin,{
       case ('All Course Videos'):
         if (paella.matterhorn && paella.matterhorn.episode && paella.matterhorn.episode.dcIsPartOf){
           var seriesId = paella.matterhorn.episode.dcIsPartOf;
-          location.href = '../ui/publicationListing.shtml?seriesId=' + seriesId;
+          // MATT-1373 reference combo page if valid DCE offering
+          if (seriesId.match('^[0-9]{11}$')) {
```

Comment:
So the series id comes in as 11 digits that we need to slice up and redirect to?